docs/reference/commandline/run.md

```diff
@@ -15,137 +15,106 @@ Aliases:
-      --device value                  Add a host device to the container (default [])
-      --device-cgroup-rule value      Add a rule to the cgroup allowed devices list
-      --device-read-bps value         Limit read rate (bytes per second) from a device (default [])
-      --device-read-iops value        Limit read rate (IO per second) from a device (default [])
-      --device-write-bps value        Limit write rate (bytes per second) to a device (default [])
-      --device-write-iops value       Limit write rate (IO per second) to a device (default [])
-      --disable-content-trust         Skip image verification (default true)
-      --dns value                     Set custom DNS servers (default [])
-      --dns-option value              Set DNS options (default [])
-      --dns-search value              Set custom DNS search domains (default [])
-      --domainname string             Container NIS domain name
-      --entrypoint string             Overwrite the default ENTRYPOINT of the image
-  -e, --env value                     Set environment variables (default [])
-      --env-file value                Read in a file of environment variables (default [])
-      --expose value                  Expose a port or a range of ports (default [])
-      --group-add value               Add additional groups to join (default [])
-      --health-cmd string             Command to run to check health
-      --health-interval duration      Time between running the check (ns|us|ms|s|m|h) (default 0s)
-      --health-retries int            Consecutive failures needed to report unhealthy
-      --health-timeout duration       Maximum time to allow one check to run (ns|us|ms|s|m|h) (default 0s)
-      --health-start-period duration  Start period for the container to initialize before counting retries towards unstable (ns|us|ms|s|m|h) (default 0s)
-      --help                          Print usage
-  -h, --hostname string               Container host name
-      --init                          Run an init inside the container that forwards signals and reaps processes
-  -i, --interactive                   Keep STDIN open even if not attached
-      --io-maxbandwidth string        Maximum IO bandwidth limit for the system drive (Windows only)
-                                      (Windows only). The format is `<number><unit>`.
-                                      Unit is optional and can be `b` (bytes per second),
-                                      `k` (kilobytes per second), `m` (megabytes per second),
-                                      or `g` (gigabytes per second). If you omit the unit,
-                                      the system uses bytes per second.
-                                      --io-maxbandwidth and --io-maxiops are mutually exclusive options.
-      --io-maxiops uint               Maximum IOps limit for the system drive (Windows only)
```

Comment:
Looks like we lost the `--io-maxbandwidth` and `--io-maxiops` options (as they're only available on Windows)